File: python/031_depositProfit.py
def depositProfit(deposit, rate, threshold):
    '''
    You have deposited a specific amount of dollars into your bank account. 
    Each year your balance increases at the same growth rate. Find out how 
    long it would take for your balance to pass a specific threshold with 
    the assumption that you don't make any additional deposits.
    '''
    current = deposit
    step = 0
    while current < threshold:
        current = current + (current * rate / 100)
        step+=1
    return step

File: arcade/go/arcade-intro/07_Through the Fog/031_depositProfit.go
func depositProfit(deposit int, rate int, threshold int) int {
	balance := float64(deposit)
	years := 0
	maxValue := float64(threshold)

	for balance < maxValue {
		years++
		balance += (float64(rate) / 100.0) * balance
		fmt.Print(balance, "\n")
	}

	return years
}
